HW04/ivan_klimov/lib/mentor.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class Mentor < Person
+  def add_homework(title:, description:, student:)
+    hw = Homework.new(title: title, description: description, student: student)
```

Comment:
Let's use full names everywhere
```suggestion
    homework = Homework.new(title: title, description: description, student: student)
```